frontend/src/App.js

```diff
@@ -1,8 +1,10 @@
 import './App.css';
 import LandingPage from './views/LandingPage'
+import News from './views/News'
+import MyNavBar from './components/Navbar'
 function App() {
   return (
-    <LandingPage/>
```

Comment:
This still has to be here. If you are having an error when you add the navbar, surround the LandingPage and Navbar in a div.